### PR TITLE
F#111 user dashboard

### PR DESCRIPTION
**Pull request by hanshank**
- Created a preferences page that is fully integrated with the backend.

- Mocked up a watchlist page that will eventually be accessed through the sidebar in the dashboard.

- Changed base_url in utils/UserApi.js/ to use .env

### CODE REVIEW

File: src/components/GenreList.js [hunk 1]
```diff
@@ -1,25 +1,123 @@
...
+      moviesByGenreKey,
+      genres,
+      style,
+      checkboxesVisible,
+      showCertifications,
+      handlePreferencesChange,
+      selectedGenres,
+      selectedCertifications,
+    } = this.props;
+    const certifications = ['PG', 'PG-13', 'R', 'G', 'NC-17'];
```
Comment (iylim) [inline src/components/GenreList.js]: rearrage certifications in order so G, PG, PG-13, R, NC-17?

File: src/components/GenreList.js [hunk 2]
```diff
@@ -1,25 +1,123 @@
...
+            <div id="list-genres">
+              {certifications.map((certification, i) => (
+                <label className="single-genre checkmark-container" key={i}>
+                  <input type="checkbox" name="certification" defaultChecked={selectedCertifications.includes(certification)} value={certification} onChange={this.handlePreferencesChange} />
+                  <span className="checkmark" />
+                  <span className="single-genre">{certification}</span>
+                </label>
+              ))}
+            </div>
+            <button className="reset-button" type="submit">reset</button>
```
Comment (iylim) [inline src/components/GenreList.js]: style reset button to look like the other buttons on our site?

File: src/components/AccountNav.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const AccountNav = () => (
+  <aside className="account-nav">
+    <ul>
+      <li>
+        <NavLink to="/account/">Movie Preferences</NavLink>
```
Comment (iylim) [inline src/components/AccountNav.js]: do we want it to be named user preferences? Its what we called the component.

File: src/components/UserPreferences.js [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+    if (pageIsLoading) {
+      return <></>;
+    }
+
+    return (
+      <div className="account-page">
+        <form className="preferences-form">
+          <div>
+            <h1 className="account-title">Movie preferences</h1>
+            <h3 className="account-sub-title">Streaming Subscriptions (enable the once you have)</h3>
```
Comment (iylim) [inline src/components/UserPreferences.js]: lowercase the S to be consistent and capitalize E in enable

File: src/components/GenreList.js [hunk 5]
```diff
@@ -1,25 +1,123 @@
...
+          
+        ))}
+      </div>
+    );
+
+    
+  }
+}
 
 GenreList.propTypes = {
```
Comment (iylim) [inline src/components/GenreList.js]: delete duplicate proptypes

File: src/components/GenreList.js (hunk 1 above)
Comment (hanshank) [reply src/components/GenreList.js]: Done!

File: src/components/UserPreferences.js (hunk 4 above)
Comment (hanshank) [reply src/components/UserPreferences.js]: Done

File: src/components/GenreList.js (hunk 5 above)
Comment (hanshank) [reply src/components/GenreList.js]: Done!

**Pull request merged**